scripts/aggregate.py

```diff
@@ -38,38 +38,30 @@ def build_summary(failures: Dict[str, List[str]], failure_name: str):
     result += "\n"
     return result
 
-def print_failed_tests(table: str):
-    print(table.split('\n'))
-    failures = table.split('\n')[2:-2]
-    allowlist: List[str] = []
-    for failure in failures:
-        print(failure.split('|'))
-        log_name = failure.split('|')[1]
-        components = log_name.split('-')
-        allowlist.append(f"{components[1]}:{components[2]}-{components[3]}")
-    print(";".join(allowlist))
-    with open("allowlist.txt", "w") as f:
-        f.write(";".join(allowlist) + "\n")
-
 def failures_to_summary(failures: Dict[str, List[str]]):
     """ Builds summary section """
     result = "# Summary\n"
     seen_failures: Set[str] = set()
-    additional_result, seen_failures = get_additional_failures("failed_build.txt",
-                                                               "Build Failures",
-                                                               seen_failures)
-    result += additional_result
-    additional_result, seen_failures = get_additional_failures("failed_testsuite.txt",
-                                                               "Testsuite Failures",
-                                                               seen_failures)
-    print_failed_tests(additional_result)
-    result += additional_result
+    build_failures, seen_failures = get_additional_failures("failed_build.txt",
+                                                            "Build Failures",
+                                                            seen_failures)
+    result += build_failures
+    testsuite_failures, seen_failures = get_additional_failures("failed_testsuite.txt",
+                                                                "Testsuite Failures",
+                                                                seen_failures)
+    result += testsuite_failures
 
     result += build_summary(failures, "New Failures")
     result += build_summary(failures, "Resolved Failures")
     result += build_summary(failures, "Unresolved Failures")
     result += "\n"
-    return result
+
+    print(result)
+
+    # Add an invalid label when no failures are detected. Something probably went wrong.
+    no_failures = not (len(build_failures) != 0 or len(testsuite_failures) != 0 or len(failures) != 0)
```

Comment:
For the readability, what do you think about `len(build_failures) == 0 and len(testsuite_failures) == 0 and len(failures) == 0`?